src/Integration.UnitTests/Binding/NuGetBindingOperationTests.cs

```diff
@@ -0,0 +1,270 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using EnvDTE;
+using FluentAssertions;
+using Microsoft.VisualStudio.ComponentModelHost;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NuGet;
+using NuGet.VisualStudio;
+using SonarLint.VisualStudio.Integration.Binding;
+using SonarLint.VisualStudio.Integration.Resources;
+using SonarQube.Client.Messages;
+
+namespace SonarLint.VisualStudio.Integration.UnitTests
+{
+    [TestClass]
+    public class NuGetBindingOperationTests
+    {
+        private ConfigurableServiceProvider serviceProvider;
+        private TestLogger logger;
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            serviceProvider = new ConfigurableServiceProvider();
+            logger = new TestLogger();
+        }
+
+        #region Tests
+
+        [TestMethod]
+        public void InstallPackages_WhenAllProjectsAreCSharp_Succeed()
```

Comment:
All of these tests were moved from _BindingWorkflowTests.cs_. I haven't added any new ones yet.